channeldb/kvdb/etcd/db.go

```diff
@@ -34,6 +38,48 @@ func newEtcdBackend(host, user, pass string) (*db, error) {
 	return backend, nil
 }
 
+// View opens a database read transaction and executes the function f with the
+// transaction passed as a parameter.  After f exits, the transaction is rolled
+// back.  If f errors, its error is returned, not a rollback error (if any
+// occur).
+func (db *db) View(f func(tx kvdb.ReadTx) error) error {
+	apply := func(stm STM) error {
+		return f(newReadWriteTx(stm))
+	}
+
+	if err := RunSTM(db.cli, apply); err != nil {
```

Comment:
style nit: can return error directly, also in `Update`

Reply:
yeah, originally I was thinking about whether or not we should do some kind of error translation.